### PR TITLE
epic(footnotes): stable target

### PR DESCRIPTION
**Pull request by ala-n**

### CODE REVIEW

File: src/modules/esl-footnotes/core/esl-note-ignore.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import {ESLMixinElement} from '../../esl-mixin-element/core';
+import {prop} from '../../esl-utils/decorators';
+
+import type {ESLNote} from './esl-note';
+
+export class ESLNoteIgnore extends ESLMixinElement {
+  public static override is = 'esl-note-ignore';
+  public static override observedAttributes: string[] = [ESLNoteIgnore.is];
```
Comment (ala-n) [inline src/modules/esl-footnotes/core/esl-note-ignore.ts:8]:
```suggestion
```

File: src/modules/esl-footnotes/core/esl-note-ignore.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import {ESLMixinElement} from '../../esl-mixin-element/core';
+import {prop} from '../../esl-utils/decorators';
+
+import type {ESLNote} from './esl-note';
+
+export class ESLNoteIgnore extends ESLMixinElement {
```
Comment (ala-n) [inline src/modules/esl-footnotes/core/esl-note-ignore.ts:6]:
```suggestion
/**
 * Optional mixin element to support esl-note-ignore attribute dynamic changes
 */
export class ESLNoteIgnore extends ESLMixinElement {
```

**Comment by ala-n** [comment]
@dshovchko sorry, but I have one more big but simple change request:
We call the feature `print` but its just one (even if a primary) case of usage that says nothing without documentation about its purpose.
It's better to rename all related technical things to exact technical features:
`anchor` (from.: `<esl-note anchor="print">` )

**Review by julia-murashko — APPROVED** [review]

**Review by NastaLeo — APPROVED** [review]

**Review by abarmina — APPROVED** [review]

**Pull request merged**